content/v2.0/get-started.md

```diff
@@ -194,6 +195,100 @@ docker exec -it influxdb /bin/bash
 {{% /tab-content %}}
 <!--------------------------------- END Docker -------------------------------->
 
+<!-------------------------------- BEGIN kubernetes---------------------------->
+{{% tab-content %}}
+
+You can install InfluxDB in a local kubernetes environment with Minikube.
+
+1. Install minikube.
+
+    On macOS, run:
+
+    ```
+    brew install minikube
+    ```
+
+2. Start minikube:
+
+    ```
+    minikube start
+    ```
+
+3. Save the following YAML configuration file somewhere on your machine:
+
+    ```yaml
+    kind: Namespace
+    apiVersion: v1
+    metadata:
+      name: influxdb
+    ---
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: influxdb
+      namespace: influxdb
+    spec:
+      selector:
+        matchLabels:
+          app: influxdb
+      template:
+        metadata:
+          labels:
+            app: influxdb
+        spec:
+          containers:
+            - name: influxdb
+              image: quay.io/influxdb/influxdb:2.0.0-alpha
+              ports:
+                - containerPort: 9999
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: influxdb
+      namespace: influxdb
+    spec:
+      type: ClusterIP
+      ports:
+        - port: 9999
+          protocol: TCP
+          name: http
+      selector:
+        app: influxdb
+    ```
+
+4. Apply the configuration by running:
+
+    ```
+    kubectl apply -f <path-to-config>.yaml
+    ```
+
+    You should see an output like this:
+
+    ```
+    namespace/influxdb configured
+    deployment.apps/influxdb configured
+    service/influxdb configured
+    ```
+
+5. Ensure the service is running:
+
+    ```
+    $ kubectl get service -n influxdb
+    NAME       TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+    influxdb   NodePort   10.100.164.55   <none>        9999:31111/TCP   26m
+    ```
+
+6. Forward the port:
+
+    ```
+    kubectl port-forward -n influxdb svc/influxdb 9999:9999 &
+    ```
+    
+Vist localhost:9999 in your browser and follow the on-screen instructions to start using InfluxDB!
```

Comment:
You can remove this. There are multiple ways to setup InfluxDB (UI and CLI) which are covered in the next section. Something you may want to cover is how to console into the running service and/or using the `influx` CLI to interact with the other service like the Docker section does above this new section.